File: models/InitializeUsers.js
const User = require("./User");

function initializeUsers() {
    const users = {};

    users["alice"] = new User("alice", "pass123");
    users["bob"] = new User("bob", "pass456");
    users["charlie"] = new User("charlie", "pass789");

    return users;
}

module.exports = initializeUsers;

File: src/main/java/org/example/library/InitializeUsers.java
package org.example.library;

public class InitializeUsers {

    public static void initialize(Library library) {
        library.addBorrower(new Borrower("alice", "pass123"));
        library.addBorrower(new Borrower("bob", "pass456"));
        library.addBorrower(new Borrower("charlie", "pass789"));
    }
}
